dev-tools/travis/travis-script.sh

```diff
@@ -24,8 +24,10 @@ cd ${STORM_SRC_ROOT_DIR}
 
 # We should be concerned that Travis CI could be very slow because it uses VM
 export STORM_TEST_TIMEOUT_MS=150000
+# Travis only has 3GB of memory, lets use 1GB for build, and 1.5GB for forked JVMs
+#export MAVEN_OPTS="-Xmx1024m"
```

Comment:
oops I should not have commented that out.  Looks like it works without it, but I meant to not have it in a comment.  will fix it.